pybliometrics/scopus/serial_search.py

```diff
@@ -0,0 +1,303 @@
+                         - 'NEUR' - Neuroscience
+                         - 'NURS' - Nursing
+                         - 'PHAR' - Pharmacology, Toxicology and Pharmaceutics
+                         - 'PHYS' - Physics and Astronomy
+                         - 'PSYC' - Psychology
+                         - 'SOCI' - Social Sciences
+                         - 'VETE' - Veterinary
+                         - 'MULT' - Multidisciplinary
+                'subjCode'
+                    Represents the SCOPUS subject area code associated with the
+                    content category desired. One or more subject codes can be
+                    provided, deliminated by comma. For full list of possible
+                    values see
+                    https://api.elsevier.com/content/subject/scopus?httpAccept=text/xml
+                'title'
+                     Represents a partial or complete serial title. Will
+                     match the pattern provided anywhere within the title -
+                     no wildcard support is provided.
+        
+        refresh : bool or int (optional, default=False)
+            Whether to refresh the cached file if it exists or not.  If int
+            is passed, cached file will be refreshed if the number of days
+            since last modification exceeds that value.
+
+        view : str (optional, default="ENHANCED")
+            The view of the file that should be downloaded.  Allowed values:
+            STANDARD, ENHANCED, CITESCORE.  For details see
+            https://dev.elsevier.com/guides/SerialTitleViews.htm.
+
+        Raises
+        ------
+        Scopus400Error
+            If provided value for a query key is invalid or for non-subscribers,
+            if the number of search results exceeds 5000.
+
+        ValueError
+            If provided query key is invalid.
+
+        Notes
+        -----
+        The directory for cached results is `{path}/{view}/{fname}`,
+        where `path` is specified in `~/.scopus/config.ini` and fname is
+        the md5-hashed version of `query` dict turned into string in format of
+        'key value' deliminated by '&'.
+        """
+        allowed_query_keys = set(['date', 'field', 'issn',
+                                  'oa', 'subj','subjCode',
+                                  'title'])
+        allowed_views = ('STANDARD', 'ENHANCED', 'CITESCORE')
+        if not set(query.keys()).issubset(allowed_query_keys):
+            raise ValueError('allowed query keys are: ' + 
+                             ', '.join(allowed_query_keys))
+        if view not in allowed_views:
+            raise ValueError('view parameter must be one of ' +
+                             ', '.join(allowed_views))
+        Search.__init__(self, query=query, api='SerialSearch',
+                        refresh=refresh, count=count, view=view)
+
+
+def _form_fields(results):
+    """Auxiliary function that returns tuple represiting union of keys
```

Comment:
Docstring contains some typos and should be of the following form:
"""Short one-line description.
empty line
Full description over multiple lines.
"""